main.go

```diff
@@ -10,6 +9,6 @@ import (
 func main() {
 	rootCmd := cmd.NewRootCommand()
 	if err := rootCmd.Execute(); err != nil {
-		fmt.Fprintln(os.Stderr, err.Error())
+		os.Exit(1)
```

Comment:
Does this mean that we're not going to print the errors anymore?

Reply:
Cobra prints the error itself. I think we should either print the error manually and set `SilenceError` attribute in `cobra.Command` or not print it and let cobra print it for us.